File: backend/src/Modules/CarRental/Domain/Values/DriverLicense.cs
namespace CarRental.Domain.Values;

public class DriverLicense
{
    public string Number { get; private set; }

    public DateTime IssuedDate { get; private set; }

    public DateTime ExpirationDate { get; private set; }

    public DriverLicense(string number, DateTime issuedDate, DateTime expirationDate)
    {
        Number = number;
        IssuedDate = issuedDate;
        ExpirationDate = expirationDate;
    }
}

File: mobile-app/app/src/main/java/com/example/easydrive/network/model/DriverLicense.java
package com.example.easydrive.network.model;

import com.google.gson.annotations.SerializedName;

public class DriverLicense {
    @SerializedName("number")
    private String number;
    
    @SerializedName("issuedDate")
    private String issuedDate;
    
    @SerializedName("expirationDate")
    private String expirationDate;
}